smf/mgs/manifest.xml

```diff
@@ -4,13 +4,18 @@
 <service_bundle type='manifest' name='mgs'>
 
 <service name='oxide/mgs' type='service' version='1'>
-  <create_default_instance enabled='false' />
+  <create_default_instance enabled='true' />
 
   <dependency name='multi_user' grouping='require_all' restart_on='none'
     type='service'>
   <service_fmri value='svc:/milestone/multi-user:default' />
   </dependency>
 
+  <dependency name='zone_network_setup' grouping='require_all' restart_on='none'
+    type='service'>
+  <service_fmri value='svc:/oxide/zone-network-setup:default' />
+  </dependency>
+
```

Comment:
I am unsure if this dependency is necessary or harmful. Could someone with more knowledge about MGS this confirm either way?

Reply:
This dependency worked fine when deploying on Madrid. Is this enough testing to guarantee that it'll be fine on a real rack also?